Fixes #482, changes design of search settings

Marauderer97 opened the pull request:
Fixes issue #482 

Changes: Design of search settings page to match the market leader in the following ways:

- [x] Adds a title

- [x] changes spacing and margins

- [x] changes font(size and family)

- [x] changes text width






Demo Link: [Here](https://marauderer97.github.io/susper.com/)

Screenshots for the change: 

Before:
![image](https://user-images.githubusercontent.com/20185076/27177099-d91bfb2e-51e2-11e7-8de5-385b7f2e03ea.png)
After:
![image](https://user-images.githubusercontent.com/20185076/27177127-ef715824-51e2-11e7-82b1-0b2fbe91ffc3.png)

harshit98 commented:
@Marauderer97 

- Do proper alignment. Compare it with market leader.
![screenshot from 2017-06-15 19-46-17](https://user-images.githubusercontent.com/22245418/27185766-f591eee6-5203-11e7-981f-89beb73652ca.png)

- Align the checkbox properly.
![screenshot from 2017-06-15 19-46-42](https://user-images.githubusercontent.com/22245418/27185786-005af962-5204-11e7-8737-c95c8c9a1db5.png)

- Design the checkbox buttons similar to Google and align it properly.
![screenshot from 2017-06-15 19-47-05](https://user-images.githubusercontent.com/22245418/27185797-0c75a850-5204-11e7-8493-258cc4f866d7.png)

- Create a `save` and `cancel` button.
![screenshot from 2017-06-15 19-47-53](https://user-images.githubusercontent.com/22245418/27185843-2c6241b4-5204-11e7-89bb-d7b61470db50.png)

harshit98 commented on src/app/searchsettings/searchsettings.component.html:
```diff
@@ -32,7 +32,8 @@
+      <div class="rightContent">
         <div>
           <h4><strong>SafeSearch Filters</strong></h4>
           <p>SafeSearch can help you block inappropriate or explicit images from your Google Search results. The SafeSearch filter isn’t 100% accurate, but it helps you avoid most violent and adult content.</p>
```
Google Search results = > Susper Search results

Marauderer97 commented:
@harshit98 Please check now. Changes I have made

- Changed the word "Google" to "Susper" everywhere

- Aligned the heading properly

- Added Save and Cancel buttons

- Designed the radio buttons following Google. In addition, have also implemented the logic that just one radio button is selected at once

- Aligned the checkbox properly

![image](https://user-images.githubusercontent.com/20185076/27214022-cafecce2-5287-11e7-8301-5eb73c09d83b.png)

Marauderer97 commented:
@harshit98 Codacy says gradients are missing for old webkit, but I have taken the css gradients from google directly, Google has not provided any gradients for the old kit, only for the new one, so I followed the same pattern
